app/src/main/java/com/x8bit/bitwarden/data/auth/repository/model/UserState.kt

```diff
@@ -91,4 +95,28 @@ data class UserState(
         val hasLoginApprovingDevice: Boolean,
         val hasResetPasswordPermission: Boolean,
     )
+
+    /**
+     * Model to encapsulate different states for a user's first time experience.
+     */
+    data class FirstTimeState(
+        val showImportLoginsCoachMarker: Boolean,
+    ) {
+        /**
+         * Constructs a [FirstTimeState] accepting nullable values. If a value is null, the default
+         * is used.
+         */
+        constructor(
+            showImportLoginsCoachMarker: Boolean?,
+        ) : this(
+            showImportLoginsCoachMarker = showImportLoginsCoachMarker ?: true,
+        )
+    }
+
+    @Suppress("UndocumentedPublicClass")
+    companion object {
+        val defaultFirstTimeState = FirstTimeState(
```

Comment:
IDK if this is something we'd want to have?

What do we gain here

Reply:
will replace with AuthDiskSource extension